cpp/src/join/distinct_hash_join.cu

```diff
@@ -205,27 +205,24 @@ CUDF_KERNEL void distinct_join_probe_kernel(Iter iter,
     cudf::size_type buffer_size = 0;
```

Comment:
*question*: I can't comment on the relevant line (above), but do you need a `flushing_tile.sync()` on line 196 after the `flush_buffer` call?

Reply:
I'm closing this issue to discuss the final sync in one place. Thanks